test/page.spec.js

```diff
@@ -0,0 +1,3812 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const fs = require('fs');
```

Comment:
Globally defined `describe` and others feel magical. Instead, can you wrap all the tests in a function and pass testrunner/defaultBrowserOptions in it as an argument?

```js
// page.spec.js
module.exports.addTests = function(testrunner, expect, defaultBrowserOptions) {
  // all the tests go here
}
```

```js
// test.js
require('page.spec.js').addTests(testrunner, defaultBrowserOptions);
```

Reply:
Yes, this is how I started, but then I thought to stay aligned with other test runners. Fixed - using `addTests` method now.